packages/audio-block/src/components/AudioPlayer.tsx

```diff
@@ -27,40 +27,43 @@ export const AudioPlayer = ({
     openFileDialog,
```

Comment:
I think it would nice to have tests for the attachments in here (also keyboard only), what do you think?

Reply:
I've added some tests but I would consider keyboard tests to be a separate pr

Reply:
Fine for me